docs/source/slurmguide/slurmrunningjobs.md

```diff
@@ -1,97 +1,67 @@
 :::
 
-This command starts an interactive bash shell on one node with one task.
-
-### Examples using `srun`
+- **`n, --ntasks=<number>`**: specify the number of tasks
+- **`N, --nodes=<minnodes[-maxnodes]>`**: specify the number of nodes
+- **`p, --partition=<partition-name>`**: specify a partition for the job to run on
 
-The user needs to review the {ref}`hardware-overview` and {ref}`partition-names` to be familiar with the available hardware and partition limits on Discovery. This way, user can tailor the request to fit both the needs of the job and the limits of partitions. For example, if the user specifies `--partition=short` and `--time=01:00:00`, it will result in an error because the time specified exceeds the limit for that partition.
+To see all options for `srun`, please refer to [srun manual] from Schedmd.
 
-This simple `srun` example is to move to a *compute* node after you first log into the HPC:
+### Examples using `srun`
 
-:::{code} bash
-srun --pty /bin/bash
-:::
+You should review the {ref}`hardware-overview` and {ref}`partition-names` pages to be familiar with the available hardware and partition limits on Discovery. This way, you can tailor the request to fit both the needs of the job and the limits of partitions. 
 
 To request one node and one task for 30 minutes with X11 forwarding on the short partition, type:
 
 :::{code} bash
-srun --partition=short --nodes=1 --ntasks=1 --x11 --mem=10G --time=00:30:00 --pty /bin/bash
+srun --partition=short --nodes=1 --ntasks=1 --x11 --mem=2G --time=00:30:00 --pty /bin/bash
 :::
 
-To request one node, with 10 tasks and 2 CPUs per task (a total of 20 CPUs), 1 GB of memory, for one hour on the express partition, type:
+To request one node, with 10 tasks and 2 CPUs per task (a total of 20 CPUs), 40 GB of memory, for one hour on the express partition, type:
 
 :::{code} bash
-srun --partition=short --nodes 1 --ntasks 10 --cpus-per-task 2 --pty --mem=1G --time=01:00:00 /bin/bash
+srun --partition=short --nodes 1 --ntasks 10 --cpus-per-task 2 --pty --mem=40G --time=01:00:00 /bin/bash
 :::
 
-To request two nodes, each with 10 tasks per node and 2 CPUs per task (a total of 40 CPUs), 1 GB of memory, for one hour on the express partition, type:
+To request 2 nodes, each with 10 tasks per node and 2 CPUs per task (a total of 40 CPUs), 80 GB of memory, for one hour on the express partition, type:
 
 :::{code} bash
-srun --partition=short --nodes=2 --ntasks 10 --cpus-per-task 2 --pty --mem=1G --time=01:00:00 /bin/bash
+srun --partition=short --nodes=2 --ntasks 10 --cpus-per-task 2 --pty --mem=80G --time=01:00:00 /bin/bash
 :::
 
 To allocate a GPU node, you should specify the `gpu` partition and use the `–gres` option:
 
 :::{code} bash
-srun --partition=gpu --nodes=1 --ntasks=1 --gres=gpu:1 --mem=1Gb --time=01:00:00 --pty /bin/bash
+srun --partition=gpu --nodes=1 --ntasks=1 --gres=gpu:1 --mem=2Gb --time=01:00:00 --pty /bin/bash
 :::
 
 (using-sbatch)=
 ## Batch Jobs: `sbatch`
```

Comment:
We should either add Command to the end of this heading, or remove Command from the Interactive Jobs heading.